docs/packages/connect/methods/getDeviceState.md

```diff
@@ -0,0 +1,44 @@
+# Get Device State
+
+Return information about a specific device
+
+```javascript
+const result = await TrezorConnect.getDeviceState(params);
+```
+
+### Params
+
+[\***\*Optional common params\*\***](commonParams.md)
+
+## Example
+
+Get the state of a device
+
+```javascript
+const capturedDevice = await TrezorConnect.getDeviceState({
+    device: {
+        instance: 0, // identificator for wallet A
+        state: undefined, // by using undefined you are making sure that passphrase will be requested
```

Comment:
lets extend parameters description a little more :)
- state: undefined => passphrase request will be enforced for instance 0 even if instance 0 is/was already using "some" pasphrase (let say empty)
Basically it works like "forget current state (if exists) and generate another one from the passphrase i will provide in the next step"
Real usecase is suite: if you add two hidden wallets with the same passphrase (but different instances) on second attempt you will receive an error about duplicated wallet and you can "try different passphrase" on current instance
 
- state: "string" => device state to verify, (state for validation) if it's valid then response will be successful, otherwise it will throw "invalid passphrase" error

Reply:
The example has more inline documentation now.